src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -77,14 +85,14 @@
 
 
         GenomeLocSortedSet includeSortedSet;
-        if (intervalStrings.isEmpty()){
+        if (getIntervalStrings().isEmpty()){
             // if -L argument isn't given but -XL is, set include to all of the territory
```

Comment:
inconsistency: the code does not check whether -XL was given. The comment says it's checked.